examples/chip-tool/BUILD.gn

```diff
@@ -21,6 +21,7 @@ assert(chip_build_tools)
 
 executable("chip-tool") {
   sources = [
+    "commands/casting/DiscoverCommissionersCommand.cpp",
```

Comment:
why not inside `/discover/` ?

Reply:
Moved it. My initial thoughts were of creating a single command cluster for all the commands needed for the  https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/rendezvous/InitiatingSetup.adoc#24-commissioner-discovery-from-an-on-network-device flow.
But I am ok with organizing this command into the pre-existing /discover/ as well. So I have moved it now.